exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/sklearn/sklearn_train_abalone.ipynb

```diff
@@ -59,7 +59,7 @@
     "stopwatch = Stopwatch()\n",
```

Comment:
<p>is there a reason you removed the link to BucketFS</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
No reason. My mistake when merging.